### DISCUSSION TITLE
Copy into Hive partition Table allow more options

### DISCUSSION THREAD

**Discussion by djouallah** [duckdb/duckdb#6071] [Ideas]
currently we have ALLOW_OVERWRITE True, which Overwrite existing partitions into a table, old partitions are not touched, that's a very useful pattern

But can we have more options something like 

ALLOW_OVERWRITE_Table True : overwrite a  whole table regardless of the partitions

ALLOW_APPEND True : append files to existing Partition ( by adding new files)

something like what Arrow dataset allow currently
![image](https://user-images.githubusercontent.com/12554469/216213100-9ba75ac8-c9a1-4e38-a310-fad7ae565f51.png)
https://arrow.apache.org/docs/python/generated/pyarrow.dataset.write_dataset.html

thanks again, it is a wonderful functionality

**Comment by samansmink** [comment]
@djouallah great suggestion! this is definitely something we want to implement soon! It's not going to make the upcoming release as we're pretty close to the release deadline unfortunately
